WeakAuras/GenericTrigger.lua

```diff
@@ -1647,7 +1647,9 @@ do
       swingTimerFrame:RegisterEvent("COMBAT_LOG_EVENT_UNFILTERED");
       swingTimerFrame:RegisterEvent("PLAYER_ENTER_COMBAT");
       swingTimerFrame:RegisterEvent("PLAYER_LEAVE_COMBAT");
-      swingTimerFrame:RegisterUnitEvent("UNIT_ATTACK_SPEED", "player");
+      if not WeakAuras.IsBCC() then
```

Comment:
But it's not in Classic _or_ Retail?

Reply:
No one complained while classic was current, so I doubt it. For retail we tested it.

Reply:
It may apply to classic too, but i'd rather not make a change i won't test